### ISSUE TITLE
0.11 Migration: `FreeSystemSet` doesn't exist anymore and isn't mentioned in the migration guide

### ISSUE THREAD

**Issue by nicopap** [bevyengine/bevy-website#724]
Just trying to deal with 3rd party code that uses `FreeSystemSet`. I suppose the correct migration instructions would be to just accept a `SystemSet` instead.

**Comment by nicopap** [comment]
Also true of the other `BaseSet` features such as `no_default_base_set`

**Comment by laundmo** [comment]
oh i JUST ran into this myself, good timing.

**Comment by nicopap** [comment]
Yeah I was trying to port bevy_spatial. Since now you have to add systems to specific schedule, you also need to accept a `schedule` parameter to your `AutoT::build` method.

**Comment by laundmo** [comment]
oh funny i was doing the same (porting bevy_spatial)
good news is i'm almost done except examples
